pyne/alara.py

```diff
@@ -843,3 +852,64 @@ def _get_subvoxel_array(mesh, cell_mats):
 
     return subvoxel_array
 
+_TO_SEC = {
+    's': 1,
+    'second': 1,
+    'm': 60,
+    'minute': 60,
+    'h': 60 * 60,
+    'hour': 60 * 60,
+    'd': 60 * 60 * 24,
+    'day': 60 * 60 * 24,
+    'w': 60 * 60 * 24 * 7,
+    'week': 60 * 60 * 24 * 7,
+    'y': 60 * 60 * 24 * 365.25,
+    'year': 60 * 60 * 24 * 365.25,
+    'c': 60 * 60 * 24 * 365.25 * 100,
+    'century': 60 * 60 * 24 * 365.25 * 100,
+}
+
+def _convert_unit_to_s(dc):
+    """
+    This function return a float number represent a time in unit of s.
+    Parameters
+    ----------
+    dc : string. Contain a num and an unit. 
+
+    Returns
+    -------
+    a float number
+    """
+    # get num and unit
+    num, unit = dc.split()
+    conv = _TO_SEC[unit]
```

Comment:
this has to be `conv = _TO_SEC.get(unit, None)` to make the following lines work when the unit is not found.

Reply:
Thanks for the comment, I have made this change in the update of this PR.